Box.V2/Converter/BoxItemConverter.cs

```diff
@@ -166,6 +166,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             // Load JObject from stream
             JObject jObject = JObject.Load(reader);
+            // The notification_email field for the user object is an object when a value exists and is an empty array when it isn't. The code below converts the array to null to avoid deserialization issues.  
+            if (jObject["type"] != null && jObject["type"].ToString() == "user" && jObject["notification_email"] != null && jObject["notification_email"].Type == JTokenType.Array)
```

Comment:
Should we check that the array is also empty, just to be sure?

Reply:
Whether it is empty or not, we should make it null or it will throw an error.